[CI] XPackRestIT test {p0=esql/60_usage/Basic ESQL usage output (telemetry) non-snapshot version} failing

**Build Scans:**
- [elasticsearch-periodic #10236 / release-tests](https://gradle-enterprise.elastic.co/s/kketplf2yxwxg)
- [elasticsearch-pull-request #87561 / checkpart3_release-tests](https://gradle-enterprise.elastic.co/s/krty52zej3duy)
- [elasticsearch-periodic #10097 / release-tests](https://gradle-enterprise.elastic.co/s/3ftpqlcuwbeqk)
- [elasticsearch-pull-request #86376 / rest-compatibility](https://gradle-enterprise.elastic.co/s/45wtnrrnlexsw)
- [elasticsearch-periodic #10084 / release-tests](https://gradle-enterprise.elastic.co/s/fgyqkkf5u2aeo)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:yamlRestTest" --tests "org.elasticsearch.xpack.test.rest.XPackRestIT.test {p0=esql/60_usage/Basic ESQL usage output (telemetry) non-snapshot version}" -Dtests.seed=B1D6286383CA74 -Dbuild.snapshot=false -Dtests.jvm.argline="-Dbuild.snapshot=false" -Dlicense.key=x-pack/license-tools/src/test/resources/public.key -Dtests.locale=hi-Deva-IN -Dtests.timezone=America/Argentina/Catamarca -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.test.rest.XPackRestIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bp0%3Desql%2F60_usage%2FBasic%20ESQL%20usage%20output%20(telemetry)%20non-snapshot%20version%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [esql/60_usage:232]: field [esql.functions] doesn't have length [139]
Expected: <139>
     but: was <140>
```

**Issue Reasons:**
- [main] 5 failures in test test {p0=esql/60_usage/Basic ESQL usage output (telemetry) non-snapshot version} (17.2% fail rate in 29 executions)
- [main] 3 failures in step release-tests (17.6% fail rate in 17 executions)
- [main] 3 failures in pipeline elasticsearch-periodic (17.6% fail rate in 17 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (16.7% fail rate in 12 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml
+++ b/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml
@@ -141,7 +141,7 @@ setup:
         - method: POST
           path: /_query
           parameters: []
-          capabilities: [ non_snapshot_test_for_telemetry, fn_month_name ]
+          capabilities: [ non_snapshot_test_for_telemetry, fn_contains ]
       reason: "Test that should only be executed on release versions"
 
   - do: {xpack.usage: {}}
@@ -229,7 +229,7 @@ setup:
   - gt: {esql.functions.to_long: $functions_to_long}
   - match: {esql.functions.coalesce: $functions_coalesce}
   - gt: {esql.functions.categorize: $functions_categorize}
-  - length: {esql.functions: 139} # check the "sister" test above for a likely update to the same esql.functions length check
+  - length: {esql.functions: 140} # check the "sister" test above for a likely update to the same esql.functions length check
 
 ---
 took:

